### PR TITLE
feat: :sparkles: Links to openfeedbacks

### LINKED ISSUE #22: [Task] Intégrer Openfeedback
## Que doit-il être fait
Permettre aux utilisateurs de faire des feedback sur les sessions, soit via Openfeedback, soit en faisant la fonctionnalité directement dans l'app.

### PR DESCRIPTION
**Pull request by gouz**
Closes #22

## Description des modifications

Add links to OpenFeedBack website on each session to allow a vote and give feedbacks to the author.

## Liste de contrôle
- [x] Je n'ai pas surdimensionné ma PR
- [x] Le titre de ma PR correspond à la [convention de commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Je n'ai pas inclus de modifications majeures
- [x] J'ai effectué ma propre relecture de code avant d'en demander une

### J'ai inclus des tests unitaires qui couvrent mes modifications
  - [X] 👍 oui
  - [ ] 🙅 non, car ils ne sont pas nécessaires
  - [ ] 🙋 non, car j'ai besoin d'aide

### J'ai ajouté/mis à jour la documentation concernant mes modifications

- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [x] 🙅 aucune documentation nécessaire

## Point technique/remarque

I think we can improve the a11y with bigger icons and a look like buttons

### CODE REVIEW

File: src/views/pages/agenda.js [hunk 1]
```diff
@@ -91,12 +92,20 @@ function sessionCard(session, isFavorite) {
...
-                                .style(isFavorite ? "rounded" : "outlined")
-                        )
-                            .onPressed(listeners.toggleFavorite, { session: session.attributes.key }),
+                            Flex(
+                                [
+                                    Actionable(
+                                        Icon("local_fire_department")
+                                            .color(isFavorite ? colors.LenraColors.yellowPulse : colors.Colors.black)
+                                            .style(isFavorite ? "rounded" : "outlined")
+                                    ).onPressed(listeners.toggleFavorite, { session: session.attributes.key }),
```
Comment (taorepoara) [inline src/views/pages/agenda.js:101]: You can add a  by `Container` with style in the `Actionnable` :
```suggestion
                                    Actionable(
                                        Container(
                                            Icon("local_fire_department")
                                                .color(isFavorite ? colors.LenraColors.yellowPulse : colors.Colors.black)
                                                .style(isFavorite ? "rounded" : "outlined")
                                        )
                                            .padding(padding.symmetric(16, 8))
                                    ).onPressed(listeners.toggleFavorite, { session: session.attributes.key }),
```

or even put them in a `Button`:

```suggestion
                                    Button("")
                                        .mainStyle("tertiary")
                                        .leftIcon(
                                            Icon("local_fire_department")
                                                    .color(isFavorite ? colors.LenraColors.yellowPulse : colors.Colors.black)
                                                    .style(isFavorite ? "rounded" : "outlined")
                                        )
                                        .onPressed(listeners.toggleFavorite, { session: session.attributes.key }),
```

File: src/views/pages/agenda.js [hunk 2]
```diff
@@ -91,12 +92,20 @@ function sessionCard(session, isFavorite) {
...
+                                    Actionable(
+                                        Icon("forum")
+                                            .color(colors.Colors.black)
+                                            .style("rounded")
+                                    ).onPressed("@lenra:navTo", { path: openfeedbacks[session.attributes.key] }),
+                                ]
+                            ).direction("vertical"),
```
Comment (taorepoara) [inline src/views/pages/agenda.js:108]: I think that an horizontal flex would be better and you should add a space between the elements and reverse there order.

```suggestion
                            ).spacing(8),
```

**Review by taorepoara — CHANGES_REQUESTED** [review]
Thanks @gouz for this PR.
You're right, the icons could be bigger and we can give them a look of a button.

For the unit tests, you can check that every session in the `camping-data.js` file has a openfeedback link.

You also should add the openfeedback link in the session pages.

**Comment by gouz** [comment]
> Thanks @gouz for this PR. You're right, the icons could be bigger and we can give them a look of a button.
> 
> For the unit tests, you can check that every session in the `camping-data.js` file has a openfeedback link.
> 
> You also should add the openfeedback link in the session pages.

All done ;) Thanks for the advices

**Review by taorepoara — APPROVED** [review]
Seems OK. Thanks again !
This PR also made me think that I forgot to add the favorite button in the session page ^^

**Pull request merged**